spin-up-hub/src/components/ContentListing.vue

```diff
@@ -32,38 +37,57 @@ export default {
                 result.map(k => {
                     if (k.score < 5) {
                         return
-                    }
+ }
                     matches.push(data.find(docs => k.ref == docs.id))
-                })
+ })
                 data = matches
-            }
+ } else {
```

Comment:
I'm guessing this is either a rogue auto-formatter or a GitHub display glitch, but every line that starts with a `}` seems to have gotten clamped to the left edge.  Could you sanity-check the formatting throughout please?